src/routes/mint/[address]/[id].svelte

```diff
@@ -1,17 +1,79 @@
 <script lang="ts">
 	import { page } from '$app/stores';
 	import { onMount } from 'svelte';
-	import * as IPFS from 'ipfs-core';
+	import { tokenUriToMetaData } from '$lib/nft-utils';
+	import { evaluateNft, getTokenUriType, getImageType } from '$lib/utils/functions';
+	import defaultAbi from '$lib/defaultAbi';
+	import { user } from '$lib/stores/user';
+	import { create } from 'ipfs-http-client'
```

Comment:
@ryanlabouve still having import issues, can you try to get a minimal ipfs-http-client working? 

import { create } from 'ipfs-http-client'

const client = create('https://ipfs.infura.io:5001/api/v0');

let results = client.add("a string that will live on IPFS"

Reply:
I think `ipfs-http-client` is the node package.

Try using the library from ipfs-core like `import { create } from 'ipfs-core'`

ref: https://github.com/ipfs-examples/js-ipfs-examples/blob/master/examples/browser-add-readable-stream/src/index.js#L3

Reply:
Side stepping with https://github.com/ryanlabouve/nftrusty-exploration/pull/28 for now